File: app/code-examples/crop-health-multi-HLT.js
const imageData = fs.readFileSync("cocoa.jpg")

// Get the multi-HLT model prediction for image cocoa.jpg
// passed as a binary file in the request body
fetch.then(async (fetch) => {
	const response_multi = await fetch(
		"https://api-test.openepi.io/crop-health/predictions/multi-HLT",
		{
			method: "POST",
			body: imageData,
		}
	)
	const data_multi = await response_multi.json()
	// Print the top 5 predictions
	console.log(data_multi)
})

File: app/code-examples/crop-health-multi-HLT.py
from httpx import Client

with Client() as client:
    # Get the multi-HLT model prediction for image cocoa.jpg 
    # passed as a binary file in the request body
    response_multi = client.post(
        url="https://api-test.openepi.io/crop-health/predictions/multi-HLT",
        content=open("cocoa.jpg", "rb").read(),
    )

    data_multi = response_multi.json()
    # Print the prediction for the MLN_maize class
    print(data_multi["MLN_maize"])
